tests/async_tests/test_interfaces.py

```diff
@@ -34,7 +34,7 @@ async def test_http_request(backend: str, server: Server) -> None:
         await read_body(stream)
 
         assert status_code == 200
-        assert ext == {"http_version": "HTTP/1.1", "reason": "OK"}
+        assert ext == {"http_version": "HTTP/1.1", "reason": ""}
```

Comment:
Interestingly, Hypercorn sends the status line as `200`, rather than a full `200 OK`, so the reason sent by the server is empty now.

Re: the follow-up to keep testing against `example.org` on 3.6, I'm not sure exactly how we'd abstract this out, but this is something we'd need to deal with.

A super simple approach would be:

```suggestion
        reason = "OK" if server.sends_reason else ""
        assert ext == {"http_version": "HTTP/1.1", "reason": reason}
```

But again, that's going to be treated as a follow-up. :)

Reply:
> But again, that's going to be treated as a follow-up. :)

Actually, given that I have to skip coverage enforcement on 3.6 so that CI keeps passing, I'm also open to doing the "fallback to a live server on 3.6" thing right in this PR, although I think doing things in 2 steps is probably easier to review, even if that creates a temporary weakness in the 3.6 test suite.